languages/tribe-ext-limit-week-view-time-range.pot

```diff
@@ -2,62 +2,137 @@
 msgid ""
 msgstr ""
 "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
-"Project-Id-Version: Events Calendar PRO Extension: Limit Week View Time "
-"Range\n"
-"POT-Creation-Date: 2018-05-18 13:59+0200\n"
-"PO-Revision-Date: 2018-03-01 09:29+0100\n"
-"Last-Translator: \n"
-"Language-Team: \n"
+"Project-Id-Version: The Events Calendar Pro Extension: Limit Week View Time "
+"Range 2.0.0\n"
+"POT-Creation-Date: 2020-10-18 00:39+0200\n"
+"PO-Revision-Date: 2020-10-17 23:14+0200\n"
+"Last-Translator: Andras Guseo <aguseo@gmail.com>\n"
```

Comment:
maybe remove yourself?

better yet:
See the WP-CLI command at https://github.com/mt-support/tribe-ext-extension-template/blob/main/README.md#good-practices to generate the POT in the best way for Extensions